fixedwidth/fixedwidth.py

```diff
@@ -221,6 +221,17 @@ def _get_decimal_data(self, field_name):
     def _get_date_data(self, field_name):
         return str(self.data[field_name].strftime(self.config[field_name]['format']))
 
+    def _format_field(self, field_name):
+        """
+        Converts field data and returns it as a string.
+        """
+        data = self.data[field_name]
+        if not data:
```

Comment:
This should be `if data is None`. Otherwise, it will return empty strings for legitimate values like `false` and `0`.